dockerfiles/ink-ci-linux/Dockerfile

```diff
@@ -44,10 +44,12 @@ RUN	set -eux; \
 # We require `xargo` so that `miri` runs properly
 # We require `grcov` for coverage reporting and `rust-covfix` to improve it.
 	cargo install grcov rust-covfix xargo; \
-# download the cargo-contracts binary tagged v0.8-ink-ci
-	curl -L "https://gitlab.parity.io/parity/cargo-contract/-/jobs/artifacts/v0.8-ink-ci/raw/artifacts/cargo-contract/cargo-contract?job=build" \
-		-o /usr/local/cargo/bin/cargo-contract; \
+# get the latest cargo-contract binary built on master
+	curl --location "https://gitlab.parity.io/parity/cargo-contract/-/jobs/artifacts/master/raw/artifacts/cargo-contract/cargo-contract?job=build" \
```

Comment:
Actually we want to have the latest released `cargo-contract` version since `master` might break assumptions from time to time.

Reply:
actually just checked, we can simply `cargo install` it